topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md

```diff
@@ -157,6 +157,7 @@ Let's start with a fresh history.
 >    A form to edit dataset attributes is displayed in the central panel
 >
 > 4. Search for `mm9` in **Database/Build** attribute and select `Mouse July 2007 (NCBI37/mm9)` (the paper tells us the peaks are from `mm9`)
+![Search in Database/Build](../../images/Search-for-mm9.PNG)
```

Comment:
```suggestion
>    ![Search in Database/Build](../../images/Search-for-mm9.PNG)
```

You need to retain the surrounding formatting, the leading `>` is important as it causes the blockquote to continue